### ISSUE TITLE
Login Window appears behind Terminal Window

### ISSUE THREAD

**Issue by al-n** [microsoft/terminal#3243]
When loging in to e.g. AzAccount, the Window where one can enter one's credentials appears behind the Windows Terminal Window and cannot be brought forward by Alt-Tabbing to it, so one has to leave the keyboard and use the mouse isntead to activate it.

![grafik](https://user-images.githubusercontent.com/9772624/67087591-84b61700-f1a3-11e9-9b02-17a832839cac.png)

**Comment by WSLUser** [comment]
I'd propose getting Alt+Tab to bring it forward for general cases when this happens, if this happens outside the Azure Shell connector, (which I think any login prompt like this one would have same issue). If it's just an Azure Shell connector specific issue , then setting the login prompt to appear in front of Terminal would be the better solution.

**Comment by al-n** [comment]
I think Azure Shell connector does not open such windows since it uses the devicelogin feature of AAD, so it is definitely not an Azure shell connector specific issue. However opening the login window in the foreground by default seems reasonable since you only see it when you invoke a command that needs it... Similar to the behavior of the window appearing when you use `Get-Credential` which already opens in foreground:
![grafik](https://user-images.githubusercontent.com/9772624/67426666-de22a980-f5da-11e9-97c4-aa0b8aed4c5a.png)

**Comment by DHowett-MSFT** [comment]
This has the same root cause as /dup #2988. Thanks for the report!
